### PR TITLE
Automation(test, Vietnam):New Script

### PR DESCRIPTION
**Pull request by Rouein**

### CODE REVIEW

**Comment by lucasrodes** [comment]
So far we have been collecting `tests performed`. Typically, this is greater than the number of people tested. However, from the [article](https://suckhoedoisong.vn/ngay-29-tet-co-12674-ca-covid-19-tai-57-tinh-thanh-giam-hon-1000-ca-so-voi-hom-qua-169220131175631105.htm):

>  Số lượng xét nghiệm từ 27/4/2021 đến nay đã thực hiện xét nghiệm được 32.204.634 mẫu tương đương 77.211.793 lượt người, tăng 21.577 mẫu so với ngày trước đó.

Google-translated into:

> The number of tests from April 27, 2021 to date has been 32,204,634 samples, equivalent to 77,211,793 turns of people, an increase of 21,577 samples compared to the previous day. 

Perhaps 77,211,793 turns of people refers to data since the beginning of the pandemic (i.e. 2020)?

**Comment by Rouein** [comment]
"turns of people" here I guess refers to people tested. I compare two consecutive days and the number of "sample tested" daily was higher than the change in "people tested".

**Comment by lucasrodes** [comment]
I see. I would assume that `people tested` then refers to cumulative value since the pandemic's beginning.

To accept this PR, I would suggest the following:

- Modify our data for Vietnam to report `people tested` instead of `tests performed`. To this end, we should the [current file](https://github.com/owid/covid-19-data/files/7973328/Vietnam-tests.csv) and manually add the data using the sources listed.
- Add this file to the `automated_sheets` folder
- Then accept this PR

Let me know your thoughts @camappel

**Comment by camappel** [comment]
We have been collecting samples (or 'mẫu' collected) tested since the beginning of the pandemic (the larger of the two 'mẫu's) from daily documents like [this one](https://luatvietnam.vn/y-te/bao-cao-149-bc-byt-2022-tinh-hinh-dich-va-ket-qua-trien-khai-cong-tac-phong-chong-dich-covid-19-tu-16h00-ngay-31-01-2022-den-16h00-ngay-01-02-2022-216476-d6.html).

However, official translations in [situation reports](https://www.who.int/vietnam/internal-publications-detail/covid-19-in-viet-nam-situation-report-78) by the WHO show that this figure actually refers to the number of RT-PCR tests performed, **including both single and pooled samples**.

In this case, the number of people tested ('lượt người được xét nghiệm') is actually greater than the number of tests ('mẫu') performed due to widespread use of pooled sampling. The daily documents state that 'the whole country has carried out 6,795,579 pooled samples for 47,926,402 turns of people'. (cả nước đã thực hiện 6,795,579 mẫu gộp cho 47,926,402 lượt người.') The pooled sampling process is explained [here](https://www.ajtmh.org/view/journals/tpmd/104/4/article-p1531.xml)

Therefore, it would be more accurate to report the number of people tested, as this is a better comparison figure for the number of cases when estimating the positive rate.

**TL;DR:** We should collect the number of people tests ('lượt người được xét nghiệm') since the start of the pandemic (**not** since 19/04/2021 ('đó từ 29/4/2021'))

image

**Review by lucasrodes — APPROVED** [review]
LGTM @camappel 

Backfilled data until 2021-09-14.

**Review by camappel — APPROVED** [review]
LGTM!
